tests/test_ops/test_points_in_polygons.py

```diff
@@ -21,3 +22,35 @@ def test_points_in_polygons():
     expected_output = torch.from_numpy(expected_output).cuda().float()
     assert torch.allclose(
         points_in_polygons(points, polygons), expected_output, 1e-3)
+
+
+@pytest.mark.parametrize('device', [
+    pytest.param(
+        'cuda',
+        marks=pytest.mark.skipif(
+            not IS_CUDA_AVAILABLE, reason='requires CUDA support')),
+    pytest.param(
+        'mlu',
+        marks=pytest.mark.skipif(
+            not IS_MLU_AVAILABLE, reason='requires MLU support')),
```

Comment:
MLU does not support this op so these lines can be removed.

Reply:
modified according to suggestions